cg/utils/utils.py

```diff
@@ -1,19 +1,13 @@
-"""Helper functions"""
+"""Helper functions."""
 
-from typing import Optional, List, Any
+from typing import Optional, List
 
 
-def get_string_from_list_by_pattern(pattern: str, collection: List[str]) -> Optional[str]:
+def get_string_from_list_by_pattern(list_of_strings: List[str], pattern: str) -> Optional[str]:
```

Comment:
```suggestion
def get_string_from_list_by_pattern(strings: List[str], pattern: str) -> Optional[str]:
```
The plural should indicate a list and also you have the type hint